src/AWSLambdaConnector.ts

```diff
@@ -303,6 +292,31 @@ export class AWSLambdaConnector extends BaseAWSConnector {
     return this.create(functionName, { filename }, options)
   }
 
+  public async createInFolder(
+    functionName: string,
+    folderHandler: (folder: Folder) => Promise<void>,
+    options: Options = {},
+  ): Promise<any> {
+    const folder = new Folder(
+      `reshuffle-awslambdaconnector-${crypto.randomBytes(8).toString('hex')}`,
+    )
+
+    try {
+      console.log(`${functionName}: Building code folder`)
```

Comment:
This and next `console.log` statements could possibly be `console.debug`

Reply:
Fixed